docs/src/install-github-ci.md

```diff
@@ -0,0 +1,37 @@
+# GitHub CI Action
+
+Kani offers a GitHub action for running Kani in the CI. As of now,
+only Ubuntu 20.04 wtih `x86_64-unknown-linux-gnu` is supported for
+Kani in the CI.
```

Comment:
1. I would say "in CI" or "the CI <something>" (where `<something> := system | workflow | pipelines | ...`, but not "in the CI".
2. "wtih" -> "with"
3. "action" -> "Action"

Reply:
9113c49ae54e848bf702b0ab31822de95f5ee3cd